packages/rancher-logging/overlay/templates/userroles.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: "logging:admin"
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups:
+      - "flows.logging.banzaicloud.io"
+      - "outputs.logging.banzaicloud.io"
+    resources:
+      - Flow
```

Comment:
Have resources be lower cased and plural. [Examples](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples)